clients/client-python/scripts/generate_version.py

```diff
@@ -0,0 +1,45 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+
+import re
+import configparser
+import subprocess
+from datetime import datetime
+
+from gravitino.constants.version import Version, VERSION_INI
+from gravitino.exceptions.gravitino_runtime_exception import GravitinoRuntimeException
+
+VERSION_PATTERN = r"version\s*=\s*['\"]([^'\"]+)['\"]"
```

Comment:
Besides, do you think we need a python function to parse the version, or a grade function is enough, like what we do for Java?

Reply:
I think python is a lot easier and more readable, and Java just get its version info using `project.version.toString()` but Python need to parse `setup.py`. 
I can refactor into Gradle function if needed.